subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseSourceSetIntegrationSpec.groovy

```diff
@@ -194,4 +194,30 @@ class EclipseSourceSetIntegrationSpec extends AbstractEclipseIntegrationSpec {
         classpath.sourceDir('src/default/java').assertOutputLocation('bin/default_')
         classpath.sourceDir('src/default_/java').assertOutputLocation('bin/default__')
     }
+
+    @ToBeFixedForConfigurationCache
+    def "Custom source set defined on dependencies"() {
```

Comment:
nit, we usually start tests with lower case letters.

```suggestion
    def "custom source set defined on dependencies"() {
```

Reply:
All the other tests here are started with upper case, might worth changing the others too.